ebsco/edsapi-koha-plugin
# Update carts for 23.11 template

#This update resolves issues with:
- download cart functionality with EDS items
- send basket functionality with EDS items
- fixes minor 23.11 issues/typos

##To test this update:
- Download the updated kpz here: [https://github.com/ebsco/edsapi-koha-plugin/blob/OM-cart-issues-2311-rework/eds_plugin_23.11001.kpz]
- Install the kpz in your existing Koha instance
- Be sure to click configure and if necessary, update your settings and save

##To test functionality of Download Cart:
- navigate to your library's Koha OPAC interface
- make sure that Discovery is selected in the search bar dropdown menu
- perform a search 
- Click 'Add to Cart' on an EDS item
- Click on your Cart in the top left corner
- Click Download in the dropdown menu, and select any download format
- Confirm that the file downloads in the format as expected
- Feel free to download multiple formats to test

##To test functionality of Send Basket:
- Ensure that your library's Koha instance is set up to send email
- navigate to your library's Koha OPAC interface
- make sure that Discovery is selected in the search bar dropdown menu
- perform a search 
- Click 'Add to Cart' on an EDS item
- Click on your Cart in the top left corner
- Click on Send
- Enter your email and proceed with sending item in the 'Sending your cart' dialogue box
- Check the email address to ensure that the message was received, with an ISO2709 attachment.

##Resolves the following issues:
#131 
#132
